src/oemof/solph/flows/_flow.py

```diff
@@ -73,10 +73,10 @@ class Flow(on.Edge):
           * `'costs``: numeric (scalar or None), the gradient cost per
             unit.
 
-    summed_max : numeric, :math:`f_{sum,max}`
+    max_capacity_factor : numeric, :math:`f_{sum,max}`
```

Comment:
Shouldn't we rename the formula element as well, e.g. to `cf_{max}`?

Reply:
Thank you, @jokochems  I forgot it.